### ISSUE TITLE
Import prompts, Modelfiles button is inactive

### ISSUE THREAD

**Issue by PaulMoney** [open-webui/open-webui#1091]
# Bug Report

## Description

**Bug Summary:**
Hi, when I click on Import prompts or Modelfiles button nothing happens (seems that they are inactive). Download buttons work correctly.

**Steps to Reproduce:**
Navigate to either Prompts or Modelfiles section and click on "Import Prompts" or " Import Modelfiles" button.

**Expected Behavior:**
Open file browser to select JSON file.

**Actual Behavior:**
Nothing happens, button is inactive.

## Environment

- **Operating System:** [Ubuntu 20.04]

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.

## Installation Method

Docker

**Comment by justinh-rahb** [comment]
I suspect I know the cause, pinging @tjbck: Is this another issue from the attempt to get rid of vanilla JS?

**Comment by tjbck** [comment]
Yes, seems to be a common theme in the PR: https://github.com/open-webui/open-webui/pull/1025/files#diff-69f3d6656db6b16bcf7427c8e7cc16c94e5a276b10fdac64e2f7a51cbfba6c70R112 Working on a fix right now.

**Comment by PaulMoney** [comment]
Confirmed, issue is resolved !
